packages/upswyng-web/src/components/BackButton.tsx

```diff
@@ -28,15 +28,23 @@ const CategoryBannerArrowBack = styled(ArrowBack)`
   }
 ` as typeof ArrowBack;
 
-const customHistory = createBrowserHistory();
+interface Props {
+  backRef?: Function;
```

Comment:
`backRef?: () => void` or something like that

Reply:
Also "ref" is a weird choice as it overloads [React refs](https://reactjs.org/docs/refs-and-the-dom.html). Maybe something like `resourceGoHome`?